From: Anand Jain <anand.jain@oracle.com>
Newsgroups: gmane.linux.file-systems,gmane.comp.file-systems.btrfs
Subject: [PATCH 1/1 V2] export symbol kobject_move()
Date: Thu, 12 Feb 2015 07:03:37 +0800
Archived-At: <http://permalink.gmane.org/gmane.linux.file-systems/92803>

drivers/cpufreq/cpufreq.c is already using this function. And now btrfs
needs it as well. export symbol kobject_move().

Signed-off-by: Anand Jain <anand.jain@oracle.com>
---
v1->v2: Didn't notice there wasn't my signed-off, now added. Thanks Dave.

 lib/kobject.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/kobject.c b/lib/kobject.c
index 58751bb..e055c06 100644
--- a/lib/kobject.c
+++ b/lib/kobject.c
@@ -548,6 +548,7 @@ out:
 	kfree(devpath);
 	return error;
 }
+EXPORT_SYMBOL_GPL(kobject_move);
 
 /**
  * kobject_del - unlink kobject from hierarchy.
-- 
2.0.0.153.g79dcccc
